src/shared/route-urls.ts

```diff
@@ -40,8 +40,15 @@ export enum RouteUrls {
   Unlock = '/unlock',
   SignatureRequest = '/signature',
 
-  // Modals
+  // Modal routes
   SignOutConfirm = 'sign-out',
   ChangeTheme = 'change-theme',
   SelectNetwork = 'choose-network',
+
+  // Send crypto asset routes
+  SendCryptoAsset = '/send-crypto-asset',
+  BitcoinSendForm = '/bitcoin-send-form',
```

Comment:
Would structure routing differently, more long the lines of:

- `/send/` → Asset select list
- `/send/btc` → Send btc form
- `/send/stx` → Send stx form

Because we don't know all possible tokens, we'd use a dynamic route: `/send/:symbol`